tests/console/perl_bootloader.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary:
+# Maintainer:
+
```

Comment:
Missing '# Summary: <multi line summary of test>' in tests/console/perl_bootloader.pm
Missing '# Maintainer: <email address>' in tests/console/perl_bootloader.pm
make: *** [test-merge] Error 1